src-ui/res/rules.ja.yaml

```diff
@@ -190,6 +190,7 @@ shikaku: "1. 盤面の点線の上に線を引き、いくつかの長方形（
 shwolf: "1. 点線の上にタテヨコに線を引いて、盤面をいくつかのブロックに分けましょう。\n2. すべてのブロックにはヤギかオオカミのどちらかを入れなければなりませんが、１つのブロックにヤギとオオカミの両方を入れてはいけません。\n 3. 線の両端は必ず盤面の外枠にしなければなりません。\n4. 線は、黒点でのみ直角に曲げることができます。なお、線を黒点で必ず曲げる必要はありませんし、線が通らない黒点があってもかまいません。\n 5. 線を交差させることはできますが、黒点では線を交差させてはいけません。"
 simpleloop: 1. 盤面の黒マス以外の全てのマスに線を引いて全体で１つの輪っかを作りましょう
 slither: "1. 盤面の点の間にタテヨコに線を引き、全体で１つの輪っかを作ります\n2. 輪っかを作る線は交差したり枝分かれしたり行き止まりがあったりしてはいけません\n 3. 数字は周りの点の間に引かれる線の数を表します。点のなかに数字の書かれていないところでは、線がいくつ引かれるかわかりません"
+swslither: "1. 盤面の点の間にタテヨコに線を引き、全体で１つの輪っかを作ります\n2. 輪っかを作る線は交差したり枝分かれしたり行き止まりがあったりしてはいけません\n 3. 数字は周りの点の間に引かれる線の数を表します。点のなかに数字の書かれていないところでは、線がいくつ引かれるかわかりません\n4. 羊はループの内側に、オオカミはループの外側にいなければなりません"
```

Comment:
Where did you get the Japanese translations?

Reply:
Google Translate.

Reply:
Having no translation is preferable over machine translation (unless it's a simple one-word translation like `inputmode.sheep` that we know to be grammatically correct)

Reply:
machine translation is actually pretty good now. especially for these kinds of simple sentences. but have removed.